app/models/digital_object/asset.rb

```diff
@@ -28,6 +29,10 @@ class DigitalObject::Asset < DigitalObject
 
   attr_accessor :skip_resource_request_callbacks
 
+  def can_have_children?
```

Comment:
Since this method is defined by the parent class with a return value of false, maybe no reason to override it here?

Reply:
I'm a little nervous of inheriting that behavior - I'm almost inclined to make the parent class method `raise` instead.

Reply:
I'm good with going with raise instead, but my other thought about this is: Should this actually be an instance method, or is static a better choice?  This is one of those cases where I wish ruby was more like java, and you could call static methods on instances.

Reply:
I'm going to break out a separate ticket for refactoring and testing `can_have_children?` and `can_have_rights?` (which the implementation was copied after): HYACINTH-826